src/plugins/ntf_json.c

```diff
@@ -456,6 +456,11 @@ srpntf_json_store(const struct lys_module *mod, const struct lyd_node *notif, co
 
             /* we are done */
             goto cleanup;
+        } else if ((notif_ts->tv_sec == from_ts) && (notif_ts->tv_sec == to_ts)) {
+            srplg_log_errinfo(&err_info, srpntf_name, NULL, SR_ERR_OPERATION_FAILED,
+                "Notification file %d-%d exceeded the maximum size of %d KB.",
```

Comment:
Please print `time_t` similarly to [other such prints](https://github.com/sysrepo/sysrepo/blob/master/src/plugins/common_json.c#L737) to avoid warnings on various platforms.